test/cmdlineTests/standard_yul_optimized/input.json

```diff
@@ -0,0 +1,23 @@
+{
+	"language": "Yul",
+	"sources":
+	{
+		"A":
+		{
+			"content": "{ let x := mload(0) sstore(add(x, 0), 0) }"
+		}
+	},
+	"settings":
+	{
+		"optimizer": {
```

Comment:
Since we might want to support different dialects, perhaps we should add a dialect setting here?
Would this be `untyped-evm`?